crates/ruff_linter/src/rules/flake8_bandit/rules/suspicious_function_call.rs

```diff
@@ -962,41 +1008,25 @@ pub(crate) fn suspicious_function_call(checker: &mut Checker, call: &ExprCall) {
             // XMLETree
             ["lxml", "etree", "parse" | "fromstring" | "RestrictedElement" | "GlobalParserTLS" | "getDefaultParser" | "check_docinfo"] => Some(SuspiciousXMLETreeUsage.into()),
             // Telnet
-            ["telnetlib", ..] => Some(SuspiciousTelnetUsage.into()),
+            ["telnetlib", _, ..] => Some(SuspiciousTelnetUsage.into()),
             // FTPLib
-            ["ftplib", ..] => Some(SuspiciousFTPLibUsage.into()),
+            ["ftplib", _, ..] => Some(SuspiciousFTPLibUsage.into()),
```

Comment:
What's the reason for updating this two match patterns?

Reply:
Thanks for providing those examples. I agree that it's unlikely that a module can be callable but it is possible. Regardless, this would mean that we won't be highlighting code like:
```py
import ftplib

getattr(ftplib, 'Error')
```

I don't expect to see this kind of usages but there's no cost to us in detecting them. So, I'd favor in reverting this change unless it's causing some other problem.

Reply:
I reverted this but feel free to comment :)